tests/consumer.test.js

```diff
@@ -315,6 +346,47 @@ const Pulsar = require('../index');
         consumer.close();
         dlqConsumer.close();
       });
+
+      test('Batch Receive', async () => {
+        const topicName = 'batch-receive-test-topic';
+        const producer = await client.createProducer({
+          topic: topicName,
+        });
+
+        const consumer = await client.subscribe({
+          topic: topicName,
+          subscription: 'sub1',
+          subscriptionType: 'Shared',
+          batchReceivePolicy: {
+            maxNumMessages: 10,
+            maxNumBytes: -1,
+            timeoutMs: 500,
```

Comment:
Could we add a case? Such as sending messages less than 10 to verify timeout works well.

Reply:
+1. You'd better test the `timeoutMs` config works.

Reply:
I will add it. 

BTW: we shouldn't need to do tests with very complete boundaries on the node client, because it depends on the CPP client, and we've added tests for it in cpp